TestYamlFileUint fails on 32 bit architectures

I'm working on packaging this library for Debian, and after enabling the altsrc package, I noticed that the test cases below fail on 32 bit architectures:

https://github.com/urfave/cli/blob/a4832fd31def973ae9fb5aedba571db952463ceb/altsrc/yaml_file_loader_test.go#L293-L302

This apples on each if `i386`, `armhf` and `armel`:

```
119s     yaml_file_loader_test.go:261: 4 testcast: expect error but Mismatched type for flag 'test'. Expected 'uint' but actual is 'int64'
119s     yaml_file_loader_test.go:261: 5 testcast: expect error but Mismatched type for flag 'test'. Expected 'uint' but actual is 'uint64'
119s --- FAIL: TestYamlFileUint (0.00s)
```

--- a/altsrc/yaml_file_loader_test.go
+++ b/altsrc/yaml_file_loader_test.go
@@ -3,7 +3,6 @@ package altsrc_test
 import (
 	"errors"
 	"fmt"
-	"io/ioutil"
 	"log"
 	"os"
 	"testing"
@@ -90,7 +89,7 @@ func ExampleApp_Run_yamlFileLoaderDuration() {
 }
 
 func TestYamlFileInt64Slice(t *testing.T) {
-	_ = ioutil.WriteFile("current.yaml", []byte(`top: 
+	_ = os.WriteFile("current.yaml", []byte(`top:
   test: [100, 9223372036854775808]`), 0666)
 	defer os.Remove("current.yaml")
 
@@ -110,7 +109,7 @@ func TestYamlFileInt64Slice(t *testing.T) {
 }
 
 func TestYamlFileStringSlice(t *testing.T) {
-	_ = ioutil.WriteFile("current.yaml", []byte(`top:
+	_ = os.WriteFile("current.yaml", []byte(`top:
   test: ["s1", "s2"]`), 0666)
 	defer os.Remove("current.yaml")
 
@@ -143,7 +142,7 @@ func TestYamlFileUint64(t *testing.T) {
 	}{
 		{
 			"top.test",
-			`top: 
+			`top:
   test: 100`,
 			false,
 		},
@@ -180,7 +179,7 @@ func TestYamlFileUint64(t *testing.T) {
 	}
 
 	for i, test := range tests {
-		_ = ioutil.WriteFile("current.yaml", []byte(test.entry), 0666)
+		_ = os.WriteFile("current.yaml", []byte(test.entry), 0666)
 		defer os.Remove("current.yaml")
 
 		testFlag := []cli.Flag{
@@ -207,7 +206,7 @@ func TestYamlFileUint(t *testing.T) {
 	}{
 		{
 			"top.test",
-			`top: 
+			`top:
   test: 100`,
 			false,
 		},
@@ -228,12 +227,12 @@ func TestYamlFileUint(t *testing.T) {
 		},
 		{
 			"test",
-			"test: 9223372036854775807", //int
+			"test: 775807", //int
 			false,
 		},
 		{
 			"test",
-			"test: 9223372036854775808", //uintt64
+			"test: 4775808", //uintt64
 			false,
 		},
 		{
@@ -244,7 +243,7 @@ func TestYamlFileUint(t *testing.T) {
 	}
 
 	for i, test := range tests {
-		_ = ioutil.WriteFile("current.yaml", []byte(test.entry), 0666)
+		_ = os.WriteFile("current.yaml", []byte(test.entry), 0666)
 		defer os.Remove("current.yaml")
 
 		testFlag := []cli.Flag{
@@ -271,7 +270,7 @@ func TestYamlFileInt64(t *testing.T) {
 	}{
 		{
 			"top.test",
-			`top: 
+			`top:
   test: 100`,
 			false,
 		},
@@ -308,7 +307,7 @@ func TestYamlFileInt64(t *testing.T) {
 	}
 
 	for i, test := range tests {
-		_ = ioutil.WriteFile("current.yaml", []byte(test.entry), 0666)
+		_ = os.WriteFile("current.yaml", []byte(test.entry), 0666)
 		defer os.Remove("current.yaml")
 
 		testFlag := []cli.Flag{

--- a/altsrc/yaml_file_loader_test.go
+++ b/altsrc/yaml_file_loader_test.go
@@ -168,7 +168,7 @@ func TestYamlFileUint64(t *testing.T) {
 		},
 		{
 			"test",
-			"test: 9223372036854775808", //uintt64
+			"test: 9223372036854775808", //uint64
 			false,
 		},
 		{
@@ -232,7 +232,7 @@ func TestYamlFileUint(t *testing.T) {
 		},
 		{
 			"test",
-			"test: 4775808", //uintt64
+			"test: 4775808", //uint64
 			false,
 		},
 		{
@@ -296,7 +296,7 @@ func TestYamlFileInt64(t *testing.T) {
 		},
 		{
 			"test",
-			"test: 9223372036854775808", //uintt64
+			"test: 9223372036854775808", //uint64
 			true,
 		},
 		{

--- a/flag_test.go
+++ b/flag_test.go
@@ -199,7 +199,7 @@ func TestFlagsFromEnv(t *testing.T) {
 		{"1", 1, &IntFlag{Name: "seconds", EnvVars: []string{"SECONDS"}}, ""},
 		{"08", 8, &IntFlag{Name: "seconds", EnvVars: []string{"SECONDS"}, Base: 10}, ""},
 		{"755", 493, &IntFlag{Name: "seconds", EnvVars: []string{"SECONDS"}, Base: 8}, ""},
-		{"deadBEEF", 3735928559, &IntFlag{Name: "seconds", EnvVars: []string{"SECONDS"}, Base: 16}, ""},
+		{"dBEEF", 900847, &IntFlag{Name: "seconds", EnvVars: []string{"SECONDS"}, Base: 16}, ""},
 		{"08", 0, &IntFlag{Name: "seconds", EnvVars: []string{"SECONDS"}, Base: 0}, `could not parse "08" as int value from environment variable "SECONDS" for flag seconds: .*`},
 		{"1.2", 0, &IntFlag{Name: "seconds", EnvVars: []string{"SECONDS"}}, `could not parse "1.2" as int value from environment variable "SECONDS" for flag seconds: .*`},
 		{"foobar", 0, &IntFlag{Name: "seconds", EnvVars: []string{"SECONDS"}}, `could not parse "foobar" as int value from environment variable "SECONDS" for flag seconds: .*`},
@@ -233,7 +233,7 @@ func TestFlagsFromEnv(t *testing.T) {
 		{"1", uint(1), &UintFlag{Name: "seconds", EnvVars: []string{"SECONDS"}}, ""},
 		{"08", uint(8), &UintFlag{Name: "seconds", EnvVars: []string{"SECONDS"}, Base: 10}, ""},
 		{"755", uint(493), &UintFlag{Name: "seconds", EnvVars: []string{"SECONDS"}, Base: 8}, ""},
-		{"deadBEEF", uint(3735928559), &UintFlag{Name: "seconds", EnvVars: []string{"SECONDS"}, Base: 16}, ""},
+		{"dBEEF", uint(900847), &UintFlag{Name: "seconds", EnvVars: []string{"SECONDS"}, Base: 16}, ""},
 		{"08", 0, &UintFlag{Name: "seconds", EnvVars: []string{"SECONDS"}, Base: 0}, `could not parse "08" as uint value from environment variable "SECONDS" for flag seconds: .*`},
 		{"1.2", 0, &UintFlag{Name: "seconds", EnvVars: []string{"SECONDS"}}, `could not parse "1.2" as uint value from environment variable "SECONDS" for flag seconds: .*`},
 		{"foobar", 0, &UintFlag{Name: "seconds", EnvVars: []string{"SECONDS"}}, `could not parse "foobar" as uint value from environment variable "SECONDS" for flag seconds: .*`},
@@ -241,7 +241,7 @@ func TestFlagsFromEnv(t *testing.T) {
 		{"1", uint64(1), &Uint64Flag{Name: "seconds", EnvVars: []string{"SECONDS"}}, ""},
 		{"08", uint64(8), &Uint64Flag{Name: "seconds", EnvVars: []string{"SECONDS"}, Base: 10}, ""},
 		{"755", uint64(493), &Uint64Flag{Name: "seconds", EnvVars: []string{"SECONDS"}, Base: 8}, ""},
-		{"deadBEEF", uint64(3735928559), &Uint64Flag{Name: "seconds", EnvVars: []string{"SECONDS"}, Base: 16}, ""},
+		{"dBEEF", uint64(900847), &Uint64Flag{Name: "seconds", EnvVars: []string{"SECONDS"}, Base: 16}, ""},
 		{"08", 0, &Uint64Flag{Name: "seconds", EnvVars: []string{"SECONDS"}, Base: 0}, `could not parse "08" as uint64 value from environment variable "SECONDS" for flag seconds: .*`},
 		{"1.2", 0, &Uint64Flag{Name: "seconds", EnvVars: []string{"SECONDS"}}, `could not parse "1.2" as uint64 value from environment variable "SECONDS" for flag seconds: .*`},
 		{"foobar", 0, &Uint64Flag{Name: "seconds", EnvVars: []string{"SECONDS"}}, `could not parse "foobar" as uint64 value from environment variable "SECONDS" for flag seconds: .*`},
@@ -1489,8 +1489,8 @@ var uintSliceFlagTests = []struct {
 }{
 	{"heads", nil, NewUintSlice(), "--heads value [ --heads value ]\t"},
 	{"H", nil, NewUintSlice(), "-H value [ -H value ]\t"},
-	{"heads", []string{"H"}, NewUintSlice(uint(2), uint(17179869184)),
-		"--heads value, -H value [ --heads value, -H value ]\t(default: 2, 17179869184)"},
+	{"heads", []string{"H"}, NewUintSlice(uint(2), uint(79869184)),
+		"--heads value, -H value [ --heads value, -H value ]\t(default: 2, 79869184)"},
 }
 
 func TestUintSliceFlagHelpOutput(t *testing.T) {

